models/websocket_chatroom.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+// WebsocketChatRoom holds information for the users inside a particular chat room
+type WebsocketChatRoom struct {
+	ID                 string           `json:"id"`                             // to hold the ID of User
```

Comment:
is this the ID of user?

Reply:
My bad. Changed it to the ID of ChatRoom